src/events/actions/updateWebsiteActions/generateVoxxrinJson.ts

```diff
@@ -0,0 +1,198 @@
+            roomsDisplayed: true
+            remindMeOnceVideosAreAvailableEnabled: true
+            showInfosTab: true
+            showRoomCapacityIndicator: false
+            hideLanguages: []
+            ratings: {
+                scale: {
+                    enabled: boolean
+                }
+                bingo: {
+                    enabled: boolean
+                }
+                'free-text': {
+                    enabled: boolean
+                }
+            }
+        }
+        sponsors: {
+            type: string
+            typeColor: string
+            typeFontColor: string
+            sponsorships: {
+                name: string
+                logoUrl: string
+                href: string
+            }[]
+        }[]
+        sessions: {
+            [key: string]: any
+        }
+        speakers: {
+            [key: string]: any
+        }
+    } = {
+        eventFamily: eventFamily,
+        title: event.name,
+        headingTitle: event.name,
+        timezone: eventTimezone,
+        start: event.dates.start?.toISOString(),
+        end: event.dates.end?.toISOString(),
+        days: days.map((day) => ({
+            id: day.start.toISO() || Math.random().toString(),
+            localDate: day.start.toLocaleString(),
+        })),
+        logoUrl: logoUrl,
+        backgroundUrl: backgroundUrl,
+        theming: {
+            colors: {
+                primaryHex: '#F78125',
+                primaryContrastHex: '#FFFFFF',
+                secondaryHex: '#3880FF',
+                secondaryContrastHex: '#FFFFFF',
+                tertiaryHex: '#202020',
+                tertiaryContrastHex: '#FFFFFF',
+            },
+        },
+        supportedTalkLanguages: supportedTalkLanguages,
+        rooms: event.tracks.map((track) => ({ id: track.id, title: track.name })),
+        talkTracks: event.categories.map((category) => ({
+            id: category.id,
+            title: category.name,
+            themeColor: category.color || '#000000',
+        })),
+        talkFormats: event.formats.map((format) => ({
+            id: format.id,
+            title: format.name,
+            duration: `PT${format.durationMinutes}m`,
+            themeColor: '#000000',
+        })),
+        features: {
+            favoritesEnabled: true,
+            roomsDisplayed: true,
+            remindMeOnceVideosAreAvailableEnabled: true,
+            showInfosTab: true,
+            showRoomCapacityIndicator: false,
+            hideLanguages: [],
+            ratings: {
+                scale: {
+                    enabled: false,
+                },
+                bingo: {
+                    enabled: false,
+                },
+                'free-text': {
+                    enabled: false,
+                },
+            },
+        },
+        sponsors: sponsors.map((sponsorCategory) => ({
+            type: sponsorCategory.name,
+            typeColor: '#000000',
```

Comment:
@HugoGresse we need different colors for `typeColor` (which should be renamed `typeBackgroundColor`) and `typeFontColor`

For defaults, I'd suggest something like `"typeColor": "#E5E4E2", "typeFontColor": "black"` but it might depend on your `sponsorCategory`.

Reply:
done!